src/utils/metadata-json-reader.ts

```diff
@@ -7,128 +7,136 @@ enum Symbolic {
   members: {
-    '__ctor__': Array<MetadataSymbol_Constructor>
-    [memberName: string]: any // TODO
-  }
+    __ctor__: Array<MetadataSymbol_Constructor>;
+    [memberName: string]: any; // TODO
+  };
   statics: {
-    [staticName: string]: MetadataSymbol_Function
-  }
+    [staticName: string]: MetadataSymbol_Function;
+  };
 }
 
 interface MetadataSymbol_Call extends MetadataSymbolBase<Symbolic.Call> {
-  expression: MetadataSymbol_Reference
+  expression: MetadataSymbol_Reference;
   arguments: Array<{
-    providers: MetadataSymbol_Reference,
-    exports: Array<MetadataSymbol_Reference>
-  }>
+    providers: MetadataSymbol_Reference;
+    exports: Array<MetadataSymbol_Reference>;
+  }>;
 }
 
-interface MetadataSymbol_Reference_External extends MetadataSymbolBase<Symbolic.Reference> {
-  module: string
-  name: string
-  line: number
-  character: number
+interface MetadataSymbol_Reference_External
+  extends MetadataSymbolBase<Symbolic.Reference> {
+  module: string;
+  name: string;
+  line: number;
+  character: number;
 }
 
-interface MetadataSymbol_Reference_Internal extends MetadataSymbolBase<Symbolic.Reference> {
-  nam: string
+interface MetadataSymbol_Reference_Internal
+  extends MetadataSymbolBase<Symbolic.Reference> {
+  nam: string;
 }
 
-type MetadataSymbol_Reference = MetadataSymbol_Reference_External | MetadataSymbol_Reference_Internal
+type MetadataSymbol_Reference =
+  | MetadataSymbol_Reference_External
+  | MetadataSymbol_Reference_Internal;
 
-interface MetadataSymbol_Constructor extends MetadataSymbolBase<Symbolic.Constructor> {
-  parameterDecorators: Array<Array<MetadataSymbol_Call>>
-  parameters: Array<MetadataSymbol_Reference>
+interface MetadataSymbol_Constructor
+  extends MetadataSymbolBase<Symbolic.Constructor> {
+  parameterDecorators: Array<Array<MetadataSymbol_Call>>;
+  parameters: Array<MetadataSymbol_Reference>;
 }
 
-interface MetadataSymbol_Function extends MetadataSymbolBase<Symbolic.Function> {
-  parameters: string[]
+interface MetadataSymbol_Function
+  extends MetadataSymbolBase<Symbolic.Function> {
+  parameters: string[];
   value: {
-    ngModule: MetadataSymbol_Reference
-    providers: Array<MetadataProvider | MetadataSymbol_Reference>
-  }
+    ngModule: MetadataSymbol_Reference;
+    providers: Array<MetadataProvider | MetadataSymbol_Reference>;
+  };
 }
 
 interface MetadataSymbol_Select extends MetadataSymbolBase<Symbolic.Select> {
-  expression: MetadataSymbol_Reference
-  member: string
+  expression: MetadataSymbol_Reference;
+  member: string;
 }
 
 type MetadataSymbol =
-  MetadataSymbol_Module |
-  MetadataSymbol_Class |
-  MetadataSymbol_Call |
-  MetadataSymbol_Reference |
-  MetadataSymbol_Constructor |
-  MetadataSymbol_Function |
-  MetadataSymbol_Select
+  | MetadataSymbol_Module
+  | MetadataSymbol_Class
+  | MetadataSymbol_Call
+  | MetadataSymbol_Reference
+  | MetadataSymbol_Constructor
+  | MetadataSymbol_Function
+  | MetadataSymbol_Select;
```

Comment:
What the fuck is this not a syntax error?

Reply:
No.